opus_codec_registration.src.html

```diff
@@ -89,6 +96,26 @@
 NOTE: Once the initialization has succeeded, any packet can be decoded at any
 time without error, but this might not result in the expected audio output.
 
+AudioEncoderConfig.contentHint guidelines {#contenthint-guidelines}
+================================================
+
+The User Agent <em class="rfc2119">MUST</em> take into consideration the value
+of {{AudioEncoderConfig/contentHint}} when configuring an Opus encoder.
+
+If {{AudioEncoderConfig/contentHint}} is <code>"[=speech=]"</code>, the User Agent
+<em class="rfc2119">MUST</em> use encoder settings that are best for most
+VoIP/videoconference applications where listening quality and intelligibility
+matter most.
+
+If {{AudioEncoderConfig/contentHint}} is <code>"[=music=]"</code>, the User Agent
+<em class="rfc2119">MUST</em> use encoder settings that are best for
+broadcast/high-fidelity applications where the decoded audio should be as close
+as possible to the input.
+
+For all other values of {{AudioEncoderConfig/contentHint}}, the User Agent
+<em class="rfc2119">MAY</em> ignore the flag or choose encoder settings based on
+any additional signals and context.
```

Comment:
This text has two issues:

- I'm not sure if we should explicitly handle the MST audio content hint `"speech-recognition"`
- Ideally, we'd force the use of `OPUS_APPLICATION_VOIP` and `OPUS_APPLICATION_AUDIO`, but those parameters appear in header files, and not in `RFC 6716` or related specs.

Reply:
The purpose of `speech-recognition` was to turn off processing like AGC. So it doesn't affect configuration like other hints might.

Reply:
Given that `OPUS_APPLICATION_VOIP` and `OPUS_APPLICATION_AUDIO` are implementation specific and not a part of the Opus spec, I don't think that this really belongs in the registry, especially if *MUST* is used to mandate vague encoder settings. 

I would make this language less forcing and put it into the main spec.

Reply:
I opened #759 instead (as to not pollute the commit history with adding & reverting commits). I am closing this PR and will address comments on that PR instead.